test/rockpapertest.js

```diff
@@ -1,16 +1,17 @@
 // IMPORT MODULES under test here:
-// import example from '../src/example.js';
+import checkResult from '../battlefunctions.js/index.js';  
```

Comment:
the `'../battlefunctions.js/index.js'` needs to be edited, because if you open your console in your browser, you'll see that there's an error: ```
GET http://localhost:5500/battlefunctions.js/index.js net::ERR_ABORTED 500 (Internal Server Error)
qunit-2.9.1.js:2745 Uncaught (in promise) Error: No tests were run.
```